src/main/java/somethingrandom/entity/DelayedItem.java

```diff
@@ -8,39 +8,31 @@
  */
 public class DelayedItem extends Item {
     private String name;
+    private UUID id;
     private final Instant creationDate;
     private Instant remindDate;
-    private final ItemKind itemKind;
 
     /**
      * Instantiates a new Delayed item.
      *
      * @param name         the name
      * @param creationDate the creation date
      * @param remindDate   the reminder date
-     * @param itemKind     the item kind
      */
-    public DelayedItem(String name, Instant creationDate, Instant remindDate, ItemKind itemKind){
+    public DelayedItem(String name, Instant creationDate, Instant remindDate){
         this.name = name;
         this.id = UUID.randomUUID();
```

Comment:
The UUID should probably be passed as a parameter so it can stay constant between runs. (We might use this to keep track of a given Item even if the program shuts down, so random UUIDs won't help.)